src/SDKs/TrafficManager/TrafficManager.Tests/ScenarioTests/EndpointScenarioTests.Features.cs

```diff
@@ -1,70 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-namespace Microsoft.Azure.Management.TrafficManager.Testing.ScenarioTests
-{
-    using System.Collections.Generic;
```

Comment:
Do you plan to delete these tests forever. They could be skipped if you intend to enable them at a future point.

Reply:
These are mostly duplicate of EndpointScnearioTests.cs. We are removing them permanently. They are not adding any extra coverage, but causing build failure due to partial definition of classes.